boxwise_wms/tests/test_random_id_sequence.py

```diff
@@ -0,0 +1,73 @@
+from odoo.tests import common
+from odoo.exceptions import Warning as UserWarning
+import logging
+import re
+
+_logger = logging.getLogger(__name__)
+
+class TestRandomIdSequence(common.TransactionCase):
+    def setUp(self):
+        super(TestRandomIdSequence, self).setUp()
+
+    @common.post_install(True)
+    def test_random_id_sequence(self):
+        sequence_model = self.env.ref("stock.seq_quant_package")
+        if not sequence_model.implementation == 'random_id':
+            sequence_model.write({'implementation': 'random_id'})
+        generatedSequence = sequence_model.next_by_code('stock.quant.package')
+
+        regexPattern = "\d{" + str(sequence_model.padding) + "}$"
+        matchingSequence = re.findall(regexPattern, generatedSequence)
+        self.assertEqual(len(matchingSequence), 1)
```

Comment:
I guess this is because the 'padding' defaults to 1? But it might not be that clear. Wouldn't the sequential sequence pass this test too?